imgui-binding/src/main/java/imgui/ImGui.java

```diff
@@ -53,10 +55,18 @@ public class ImGui {
 
         if (libPath != null) {
             System.load(Paths.get(libPath).resolve(fullLibName).toFile().getAbsolutePath());
-        } else if (extractedLibAbsPath != null) {
-            System.load(extractedLibAbsPath);
         } else {
-            System.loadLibrary(libName);
+            try {
+                System.loadLibrary(libName);
+            } catch (Exception | Error e) {
+                if (extractedLibAbsPath != null) {
```

Comment:
If we're loading extracted library here - library extraction should be moved here as well. So it won't be like we extracted library from classpath, but loaded library from ENV var and extraction did nothing.